<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1414</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YggTorrent Downloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: rgb(0, 0, 0);
selection-background-color: rgb(0, 255, 0);
selection-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);
}
QMenu
{
background-color: rgb(0, 0, 0);
selection-background-color: rgb(0, 255, 0);
selection-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);
}
QMenu:hover
{
background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);
}
QTableView
{
	border: 0px;
	background-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
	gridline-color: rgb(53, 53, 53);
}
QTableView:section
{
	background-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
}
QHeaderView:section
{
	background-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
}
QTableView:disabled
{
	background-color: rgb(20, 20, 20);
	color: rgb(0,0,0);
}
QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;    
	background-color: rgb(0, 0, 0);
    min-width: 80px;	
	color: rgb(0, 255, 0);
}

QPushButton:hover {
   
	background-color: rgb(0, 255, 0);
	color: rgb(0, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
}
QPushButton:disabled{
	background-color: rgb(25, 25, 25);
	color: rgb(0, 255, 0);
    border: 0px solid #8f8f91;
}
QComboBox{
	background-color: rgb(0, 0, 0);
	selection-background-color: rgb(0, 255, 0);
	selection-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
}
QLineEdit
{
	border: 0px;
	background-color: rgb(0, 0, 0);
	selection-background-color: rgb(0, 255, 0);
	selection-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
}
QLabel
{
	background-color: rgb(0, 0, 0);
	selection-background-color: rgb(0, 255, 0);
	selection-color: rgb(0, 0, 0);
	color: rgb(0, 255, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="eFind">
        <property name="placeholderText">
         <string>Recherche</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bFind">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="top400">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Top 400</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Tout</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>film</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>musique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>application</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ebook</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jeux video</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reload">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>  Recharger Téléchargement  </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Téléchargement en cours :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="downCount">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="result">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTableWidget" name="listTorrent">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Age</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Taille</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Seeder</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Lien</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1414</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuStyle">
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionLight"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuStyle"/>
    <addaction name="actionOptions"/>
    <addaction name="actionA_Propos"/>
    <addaction name="actionA_Propos_de_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_Propos">
   <property name="text">
    <string>A Propos</string>
   </property>
  </action>
  <action name="actionA_Propos_de_Qt">
   <property name="text">
    <string>A Propos de Qt</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>eFind</sender>
   <signal>returnPressed()</signal>
   <receiver>bFind</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>1359</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
